src/ucp/wireup/wireup_cm.c

```diff
@@ -515,6 +512,7 @@ static unsigned ucp_cm_client_uct_connect_progress(void *arg)
         goto try_fallback;
     }
 
+    ucp_wireup_replay_pending_requests(ep, &tmp_pending_queue);
```

Comment:
shouldn't we do it after line 520?
@evgeny-leksikov WDYT?

Reply:
👍 that would be better aligned with initial ep_create flow